File: myonlinefm/src/main/java/com/sec/myonlinefm/data/ClassificationAttributePattern.java
package com.sec.myonlinefm.data;

import java.util.HashMap;
import java.util.List;

/**
 * Created by SRC-TJ-MM-BinYang on 2018/4/19.
 */

public class ClassificationAttributePattern {
    private List<PropertyInfo> mInfo = null;
    private HashMap<Integer, List<PropertyInfo.values>> mInfoMap = null;

    public void setInfo(List<PropertyInfo> mInfo) {
        this.mInfo = mInfo;
    }
    public List<PropertyInfo> getInfo() {
        return mInfo;
    }

    public void setInfoMap(HashMap<Integer, List<PropertyInfo.values>> mInfoMap) {
        this.mInfoMap = mInfoMap;
    }
    public HashMap<Integer, List<PropertyInfo.values>> getInfoMap() {
        return mInfoMap;
    }
}

File: myonlinefm/src/main/java/com/sec/myonlinefm/data/ClassificationAttributePattern.kt
package com.sec.myonlinefm.data

/**
 * Created by SRC-TJ-MM-BinYang on 2018/4/19.
 */
class ClassificationAttributePattern {
    private var mInfo: MutableList<PropertyInfo?>? = null
    private var mInfoMap: HashMap<Int?, MutableList<PropertyInfo.values?>?>? = null
    fun setInfo(mInfo: MutableList<PropertyInfo?>?) {
        this.mInfo = mInfo
    }

    fun getInfo(): MutableList<PropertyInfo?>? {
        return mInfo
    }

    fun setInfoMap(mInfoMap: HashMap<Int?, MutableList<PropertyInfo.values?>?>?) {
        this.mInfoMap = mInfoMap
    }

    fun getInfoMap(): HashMap<Int?, MutableList<PropertyInfo.values?>?>? {
        return mInfoMap
    }
}